README.md

```diff
@@ -139,16 +145,22 @@ To configure the Slack operator in your local instance, from the Airflow UI go t
 
 - 🐚 get a shell on a worker, for example
 
-```
-docker exec -it airflow-airflow-worker-1 bash
+```shell
+$ docker exec -it airflow-airflow-worker-1 bash
```

Comment:
🐚 Nice additions, thank you - helpful indications of how to use things are really good.